rules/review-automated-tests/rule.md

```diff
@@ -76,3 +82,6 @@ This is where regular review can help us, as Michael notes in his blog post, [On
 When we have unjustified trust in our checks, we have the opposite problem that we have with the smoke detector: we're unlikely to notice that the alarm doesn’t go off when it should.
 
 > We can choose to hold on to the possibility that something might be wrong with our checks, and to identify the absence of red checks as meta-information; a suspicious silence, instead of a comforting one. The responsible homeowner checks the batteries on the smoke alarm."
+
+## Further reading
+
```

Comment:
Hey @leehawkinsssw 

I think the "further reading" links are missing.

Can you fix?

Reply:
Mea culpa, that section should be removed. I was going to add a reference but ended up just using a couple of ideas from it for the edits I made.

Reply:
done - I removed that section